### PR TITLE
Removed hard-coded support email

### PR DESCRIPTION
**Pull request by orangejenny**
Minor.

### CODE REVIEW

**Review by proteusvacuum — APPROVED** [review]

**Review by mkangia — APPROVED** [review]
Is there a corresponding PR in `commcare-cloud` to add this in settings?

File: corehq/apps/hqadmin/tasks.py [hunk 1]
```diff
@@ -142,7 +139,7 @@ def send_abnormal_usage_alert(alert):
     )
     send_html_email_async(
         subject,
-        support_email,
+        settings.SUPPORT_EMAIL,
```
Comment (mkangia) [inline corehq/apps/hqadmin/tasks.py]: what would happen if there is no email set in settings?

**Comment by orangejenny** [comment]
@mkangia `SUPPORT_EMAIL` is already in settings.

File: corehq/apps/hqadmin/tasks.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/hqadmin/tasks.py]: Hmm, I don't know. There are default values for all of the emails in settings, so I'm not worried about it.

Comment (mkangia) [reply corehq/apps/hqadmin/tasks.py]: Sounds good, thanks for confirming

**Pull request merged**